Description: Fixing spelling errors in the source files
 Fixed 2 spelling errors in the source files

Forwarded: Yes
Author: Elías Alejandro Año Mendoza <ealmdz@gmail.com>
Last-Update: 2017-09-09

--- uget.orig/ui-gtk/UgtkSettingForm.c
+++ uget/ui-gtk/UgtkSettingForm.c
@@ -361,7 +361,7 @@ void  ugtk_completion_form_init (struct
 
 	hbox = (GtkBox*) gtk_box_new (GTK_ORIENTATION_HORIZONTAL, 0);
 	gtk_box_pack_start (vbox, (GtkWidget*) hbox, FALSE, FALSE, 1);
-	widget = gtk_label_new (_("Custom command if error occured:"));
+	widget = gtk_label_new (_("Custom command if error occurred:"));
 	gtk_box_pack_start (hbox, widget, FALSE, FALSE, 0);
 
 	entry = gtk_entry_new ();

--- uget.orig/uget/UgetPluginAria2.c
+++ uget/uget/UgetPluginAria2.c
@@ -280,7 +280,7 @@ static const char* error_string[] =
 	N_("aria2: an unknown error occurred."),
 	N_("aria2: time out occurred."),
 	N_("aria2: resource was not found."),
-	N_("aria2 saw the specfied number of 'resource not found' error. See --max-file-not-found option"),
+	N_("aria2 saw the specified number of 'resource not found' error. See --max-file-not-found option"),
 	N_("aria2: speed was too slow."),
 	N_("aria2: network problem occurred."),
 	N_("aria2: unfinished downloads."),
